### PR TITLE
Make web-library-build export the tasks it defines.

### PR DESCRIPTION
**Pull request by cliffkoh**
Make web-library-build export the tasks it defines.

### CODE REVIEW

File: web-library-build/src/index.ts [hunk 1]
```diff
@@ -34,30 +34,33 @@ const sourceMatch: string[] = [
 ];
 
 // Define default task groups.
-let buildTasks: IExecutable = task('build', serial(preCopy, sass, parallel(tslint, typescript, text), postCopy));
```
Comment (cliffkoh) [inline web-library-build/src/index.ts]: I actually have no idea what preCopy and postCopy does (line 26-29)...

Comment (dzearing) [inline web-library-build/src/index.ts]: they allow you to specify a map of dest: [sources] to copy from one place to another. Sometimes you need that to be done before all the building, sometimes after.

File: web-library-build/src/index.ts [hunk 2]
```diff
@@ -34,30 +34,33 @@ const sourceMatch: string[] = [
...
-let bundleTasks: IExecutable = task('bundle', serial(buildTasks, webpack));
-const postProcessSourceMaps: PostProcessSourceMaps = new PostProcessSourceMaps();
-const validateShrinkwrapTask: ValidateShrinkwrapTask = new ValidateShrinkwrapTask();
-const generateShrinkwrapTask: GenerateShrinkwrapTask = new GenerateShrinkwrapTask();
+export const buildTasks: IExecutable = task('build', serial(preCopy, sass, parallel(tslint, typescript, text), postCopy));
```
Comment (dzearing) [inline web-library-build/src/index.ts]: should (typescript, text) be compileTsTasks?

File: web-library-build/src/index.ts [hunk 3]
```diff
@@ -34,30 +34,33 @@ const sourceMatch: string[] = [
...
+export const bundleTasks: IExecutable = task('bundle', serial(buildTasks, webpack));
+export const compileTsTasks: IExecutable = parallel(typescript, text);
+export const testTasks: IExecutable = serial(sass, compileTsTasks, karma);
+export const defaultBuildBundleAndTest: IExecutable = serial(bundleTasks, karma);
```
Comment (dzearing) [inline web-library-build/src/index.ts]: defaultTasks?

File: web-library-build/src/index.ts (hunk 1 above)
Comment (cliffkoh) [inline web-library-build/src/index.ts]: :+1: also augmented documentation of copy task.

File: web-library-build/src/index.ts (hunk 3 above)
Comment (cliffkoh) [inline web-library-build/src/index.ts]: Just wanted to make it more explicit what default does, renamed it to `buildBundleAndTestDefaultTask` to make it more apparent.

File: web-library-build/src/index.ts (hunk 2 above)
Comment (dzearing) [inline web-library-build/src/index.ts]: i'm saying, define it above this and actually use it so you don't duplicate the definition.

**Pull request merged**